pmlc/compiler/executable.cc

```diff
@@ -164,7 +164,19 @@ static void *tryResolveSymbol(StringRef symbol) {
     if (auto ptr = resolveSymbol(symbol.drop_front()))
       return ptr;
   }
-  return llvm::sys::DynamicLibrary::SearchForAddressOfSymbol(symbol.str());
+
```

Comment:
Isn't this already handled in the few lines above?

Reply:
The code just above handles resolving symbols from our library. There is no code to handle system library symbols like llvm::sys::DynamicLibrary::SearchForAddressOfSymbol("_cos"); for example. I merely added code to handle this case (as it was in the Stripe version).